superset/security/manager.py

```diff
@@ -145,9 +145,9 @@ def __init__(self, **kwargs: Any) -> None:
 RoleModelView.related_views = []
 
 
-def freeze_metric(metric: Metric) -> str:
+def freeze_value(metric: Metric) -> str:
```

Comment:
Super nitpicky, but maybe we want to also change the parameter name from `metric` to something generic as well (such as `value`)?

Reply:
D'oh, good point! I missed that!